app/assets/javascripts/admin/addon/components/schema-theme-setting/types/models.gjs

```diff
@@ -25,18 +23,14 @@ export default class SchemaThemeSettingTypeModels extends Component {
   }
 
   get validationErrorMessage() {
-    if (!this.touched) {
-      return;
-    }
```

Comment:
I decided to remove this and display the validation message upfront since I feel like it'll be clearer.